### PR TITLE
Execute multiple queries in macro

### PR DESCRIPTION
**Pull request by StanislasGuinel**
[CH62564] Multiple queries can be executed inside the same transaction commit in the macro (queries must be separated by a semicolon like in .cypher files)

### CODE REVIEW

File: python-runnables/neo4j-macro/runnable.json [hunk 1]
```diff
@@ -15,14 +15,14 @@
...
+            "parameterSetId": "neo4j-server-configuration",
+            "mandatory": true
         },
         {
             "name": "cypherQuery",
-            "label": "Query",
+            "label": "Cypher queries",
             "type": "TEXTAREA",
-            "description": "Type your custom Cypher query",
+            "description": "Type your custom Cypher queries. Multiple queries are separated with a ';'.",
```
Comment (alexbourret) [inline python-runnables/neo4j-macro/runnable.json:25]: Do you want to mention the fact that this is one transaction here ?

File: python-runnables/neo4j-macro/runnable.py [hunk 2]
```diff
@@ -1,40 +1,67 @@
...
 
-        html = html + "<h5>Query results (if any, truncated to 50)</h5>"
-        try:
-            df = r.to_data_frame()
-            html = html + df.to_html(index=False, max_rows=50)
-        except Exception as e:
-            html = html + "<pre>No results to display. Error: {}</pre>".format(e)
+        html += "<h5>Queries data (if any, truncated to first 10 records}</h5>"
+
+        for i, query in enumerate(queries):
```
Comment (alexbourret) [inline python-runnables/neo4j-macro/runnable.py:58]: i ):[

**Review by alexbourret — APPROVED** [review]

File: python-runnables/neo4j-macro/runnable.py (hunk 2 above)
Comment (StanislasGuinel) [reply python-runnables/neo4j-macro/runnable.py]: 730e68a

File: python-runnables/neo4j-macro/runnable.json (hunk 1 above)
Comment (StanislasGuinel) [reply python-runnables/neo4j-macro/runnable.json]: 730e68a

**Pull request merged**